README.md

```diff
@@ -125,46 +82,49 @@ environment variables.  See the
 1. Pull the new image:
 
     ```console
-    docker pull cisagov/example:0.0.1
+    docker pull cisagov/guacscanner:0.0.1-rc.18
     ```
 
-1. Recreate and run the container by following the [previous instructions](#running-with-docker).
+1. Recreate and run the container by following the [previous
+   instructions](#running-with-docker).
 
 ## Image tags ##
 
 The images of this container are tagged with [semantic
-versions](https://semver.org) of the underlying example project that they
-containerize.  It is recommended that most users use a version tag (e.g.
-`:0.0.1`).
+versions](https://semver.org) of the underlying example project that
+they containerize.  It is recommended that most users use a version
+tag (e.g. `:0.0.1-rc.18`).
 
 | Image:tag | Description |
 |-----------|-------------|
-|`cisagov/example:1.2.3`| An exact release version. |
-|`cisagov/example:1.2`| The most recent release matching the major and minor version numbers. |
-|`cisagov/example:1`| The most recent release matching the major version number. |
-|`cisagov/example:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
-|`cisagov/example:nightly` | A nightly build of the `develop` branch of this repository. |
-|`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
+|`cisagov/guacscanner:1.2.3`| An exact release version. |
+|`cisagov/guacscanner:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/guacscanner:1`| The most recent release matching the major version number. |
```

Comment:
Match current version:
```suggestion
|`cisagov/guacscanner:0.0.1`| An exact release version. |
|`cisagov/guacscanner:0.0`| The most recent release matching the major and minor version numbers. |
|`cisagov/guacscanner:0`| The most recent release matching the major version number. |
```

Reply:
I know we do this elsewhere, but looking at it this time I think the default values (because the major, minor, and patch values are all different from one another) make the meaning behind each of the the tags clearer.  If the version is `0.0.1`, and you check out `cisagov/guacscanner:0`, it isn't clear a priori that the zero in the image tag refers to the major version.

This is a subtle point, but I think it is an important one.

Reply:
I see what you're saying, but I still think that this is clear when coupled with the corresponding explanatory description text.  I like always showing in the docs what the current versions are, rather than tags that may or may not exist or be the latest versions.  I think we need @mcdonnnj and @felddy to weigh in here.

Reply:
I'm not convinced it's best, but I will defer to the inertia of the way we have been doing it.   Please enjoy commit 2446f05.

Reply:
Maybe open an issue or discussion for this in skeleton-docker?